arelle/plugin/validate/EDINET/rules/gfm.py

```diff
@@ -647,6 +647,43 @@ def rule_gfm_1_3_1(
         )
 
 
+@validation(
+    hook=ValidationHook.XBRL_FINALLY,
+    disclosureSystems=[DISCLOSURE_SYSTEM_EDINET],
+)
+def rule_gfm_1_3_2(
+        pluginData: PluginValidationDataExtension,
+        val: ValidateXbrl,
+        *args: Any,
+        **kwargs: Any,
+) -> Iterable[Validation]:
+    """
+    EDINET.EC5700W: [GFM 1.3.2] The schemaLocation attribute of the xsd:import element specifying the EDINET taxonomy does not point to a valid location.
+
+    GFM-1.3.2:  If an xsd:import element has a namespace attribute equal to a standard taxonomy
+    schema, then its schemaLocation attribute must be the standard taxonomy assigned
+    to that namespace.
+    """
+    for document in val.modelXbrl.urlDocs.values():
+        if pluginData.isExtensionUri(document.uri, val.modelXbrl) and document.type == ModelDocument.Type.SCHEMA:
+            for refDoc in document.referencesDocument.values():
+                if 'import' in refDoc.referenceTypes:
+                    namespace = refDoc.referringModelObject.attrib.get('namespace')
+                    if namespace:
+                        if namespace.endswith('iod'):
```

Comment:
It's not clear to me why this is necessary. Is there something EDINET-specific going on here?

Reply:
Could we use `Namespace` values in `edinet-taxonomies.xml` to determine when "an xsd:import element has a namespace attribute equal to a standard taxonomy schema", and then compare the `schemaLocation` to `Href`?

Reply:
It is specific to EDINET and the Identification and Ordering namespace.  The namespace as imported is: `http://disclosure.edinet-fsa.go.jp/taxonomy/common/2013-08-31/iod`. The namespace for the XSD is  `http://disclosure.edinet-fsa.go.jp/taxonomy/common/2013-08-31/identificationAndOrdering_2013-08-31.xsd`

Reply:
I don't love relying on the pattern that the schema location URL _starts with_ the corresponding namespace URI, and then needing to hardcode in the exceptions to that rule. Ideally these implementations would be portable to other jurisdictions. We have this existing design around "standard taxonomies" defined in the XML that I think we can make use of.